server/helpers/userValidator.js

```diff
@@ -0,0 +1,176 @@
+import { Pool } from 'pg';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const env = process.env.NODE_ENV;
+let connectionString;
+
+if (env === 'development') {
+  connectionString = process.env.DATABASE_URL;
+} else {
+  connectionString = process.env.use_env_variable;
+}
+
+const pool = new Pool({
+  connectionString,
+});
+
+export const verrifyUserExist = (req, res, next) => {
+  const email = req.body.email.toLowerCase();
+  const queryValues = [];
+  const error = {};
+  let errorChecker = false;
+
+  queryValues.push(email);
+
+  pool.query('SELECT * FROM users WHERE email = $1', [queryValues[0]], (err, result) => {
+    if (err) {
+      errorChecker = true;
+      error.err = err;
+    }
+    if (result.rows.length > 0) {
+      errorChecker = true;
+      error.user = 'This email/user already exist in our server, try signing in.';
```

Comment:
`error.user` insinuates that one is about to attach a user object to the `error` field. Could you use a variable name that insinuates what you're inserting is a message?